runtime/Cargo.toml

```diff
@@ -85,6 +85,7 @@ traced-test = { workspace = true }
 
 [features]
 default = ['std']
+fast-runtime = []
```

Comment:
We should probably add a mention of this to the README explaining the development intent. It would be useful for anyone new coming into the project.

Reply:
@nathanwhit , @AdaJane HEADS UP: I've rebased in order to bump the version b/c CI wasn't happy. 

Also added a comment in this file to describe this feature. I didn't see if we have all of the other features documented somewhere so opted for the minimum approach.